### ISSUE TITLE
CloudWatch: DataAlreadyAcceptedException

### ISSUE THREAD

**Issue by bobzoller** [segmentio/ecs-logs#68]
Occasionally we get a `DataAlreadyAcceptedException` when writing to CloudWatch.  When we see this error, it is followed by some number of `the writer was invalidated by another goroutine`, each of which represents some number of dropped log lines.

Until now, it was happening infrequently enough (could go weeks without an occurrence) that we didn't worry about it.  In the last 5 days, however, it's been consistently happening 10-30 times per day (total across all our log streams), so now I'm digging into it.

Can anyone help me understand:
- why is this happening?
- how can we address it so we stop dropping these log batches?

It looks like this error could potentially be handled gracefully in the same way [`InvalidSequenceTokenException` is already](https://github.com/segmentio/ecs-logs/blob/master/lib/cloudwatchlogs/writer.go#L71-L79), because the error message includes a new token:

```
DataAlreadyAcceptedException: The given batch of log events has already been accepted. The next batch can be sent with sequenceToken: 49586195678550359962035101545865519144132306138323654946
```

**Comment by bobzoller** [comment]
I should mention we're running 1b0bfee9b6c0357c902141673f3a2050d63dff90 (so do not have the fixes in #67 or #65)

**Comment by bobzoller** [comment]
I think amekkawi/cwlogs-writable#10 confirmed my theory and I'm feeling like I will just take a pass at a PR.
